pkg/component/watch_test.go

```diff
@@ -1,3 +1,6 @@
+// +build !windows
+// +build !osx
```

Comment:
https://github.com/openshift/odo/pull/2562/files#r388802204

Reply:
@prietyc123 Can we make this in a single line ? For example 

```
// +build !windows !osx
```

Reply:
> @prietyc123 Can we make this in a single line ? For example
> 
> ```
> // +build !windows !osx
> ```

Done